src/test/java/api/support/APITests.java

```diff
@@ -99,6 +99,7 @@
   protected final ResourceClient instancesClient = ResourceClient.forInstances();
   protected final ResourceClient holdingsClient = ResourceClient.forHoldings();
   protected final ResourceClient itemsClient = ResourceClient.forItems();
+  protected final ResourceClient itemsLostRequiringActualCostsClient = ResourceClient.forItemsLostRequiringActualCosts();
```

Comment:
Could this be created only for the test classes that need it rather than the shared base class?